### PR TITLE
Reorganize unclear dependencies  [PD-1739]

### PR DESCRIPTION
**Pull request by darrint**
This doesn't remove circular dependencies, but clarifies some mystery situations. It should be easier to fix up circular deps in the future.

### CODE REVIEW

File: seo/admin.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
                         GoogleAnalyticsCampaign, SeoSite, SeoSiteFacet,
                         SeoSiteRedirect, SiteTag, SpecialCommitment, ViewSource)
 from seo.queryset_copier import copy_following_relationships
-from seo.signals import check_message_queue
+from seo.models import check_message_queue
```
Comment (aspidites) [inline seo/admin.py]: Doesn't this make things more confusing, considering check_message_queue is defined in seo.signals, not seo.models?

**Comment by aspidites** [comment]
I feel like this PR is doing more than advertised. In particular, the changes to seo.models seems to be about more than simply reorganizing dependencies.

**Comment by darrint** [comment]
To move the signals import I have to do what I did, moving it to the models code as Troy's links point out. And yes, for 1.8 there are new and better options available.

File: seo/admin.py (hunk 1 above)
Comment (aspidites) [inline seo/admin.py]: Disregard. Now that it isn't 5pm, this makes sense.

**Comment by JLMcLaugh** [comment]
Merge conflicts

**Pull request merged**